### PR TITLE
Add Brunt cover docs

### PR DESCRIPTION
**Pull request by eavanvalkenburg**
**Description:**
Added docs for the Brunt cover component.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#15653

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/cover.brunt.markdown [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+title: Brunt Cover
+description: "Instructions on how to set up Brunt Blind Engine within Home Assistant."
+date: 2018-06-24 14:36
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: brunt.png
+ha_category: Cover
+ha_release: 0.69
```
Comment (frenck) [inline source/_components/cover.brunt.markdown]: `0.69` has already shipped a while ago, please update this to the next major release; `0.73`

File: source/_components/cover.brunt.markdown [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+title: Brunt Cover
+description: "Instructions on how to set up Brunt Blind Engine within Home Assistant."
+date: 2018-06-24 14:36
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: brunt.png
```
Comment (frenck) [inline source/_components/cover.brunt.markdown:10]: This logo does not exist in our repository.
Please add it to the PR.

**Review by frenck — CHANGES_REQUESTED** [review]
Hi, @eavanvalkenburg.
I've reviewed the documentation PR and left 2 comments.
Could you please take a look? 👍

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Thank you, @eavanvalkenburg, for updating the PR.
✅ Approved and can be merged as soon as the parent PR gets merged.

**Comment by eavanvalkenburg** [comment]
Thanks for reviewing @frenck!

**Review by frenck — DISMISSED** [review]

File: source/_components/cover.brunt.markdown [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+ha_category: Cover
+ha_release: 0.75
+ha_iot_class: "Cloud Polling"
+---
+
+The `brunt` platform allows one to control Blind Engines by [Brunt](https://www.brunt.co). To use this sensor, you need a Brunt App Account. Automatically all Brunt devices registered to your account are added.
+
+## {% linkable_title Configuration %}
+
+To enable this sensor, add the following lines to your `configuration.yaml`:
```
Comment (MartinHjelmare) [inline source/_components/cover.brunt.markdown]: To enable this integration

**Review by MartinHjelmare — APPROVED** [review]

**Comment by MartinHjelmare** [comment]
Can be merged when build passes.

**Pull request merged**